### ISSUE TITLE
Depended fields check box can't be unchecked. 

### ISSUE THREAD

**Issue by ivandotv** [vafour/vafpress-framework#13]
I'm test driving the latest version of this framework in combination with vafpress-framework-theme-boilerplate
And looking at the theme options -> field interactions->depended field 
After selecting the font, and checking `font weight` or `font style` check box fields, I'm unable to un-check them.

**Comment by ivandotv** [comment]
Not to open another issue , but the same thing happens in  "Advanced usage -> typography with a preview"
After clicking the check boxes I can't uncheck them, however the preview is updating but without any visual differences  ( there is no difference between font weights and styles in the preview).

**Comment by daviedR** [comment]
HI ikixxx,

For Field Interactions > Dependent Field, we have `font weight` and `font style` as radiobuttons, not as checkboxes. It doesn't work like checkboxes where you can check and uncheck them. If you want that kind of manner, you can change the field's `type` of the `font weight` and `font style` fields into `checkbox` (instead of `radiobutton`). You can find it at `admin\option\option.php`

After you select a `font family`, the `font weight` and `font style` data pool will be updated and resetted with no choice selected. Since no style and weight being defined, the preview will show the default weight and style, which is `normal`. So if you select `normal` on the `font weight` or `font style` it won't show any differences to the default one. `normal` = default. If you select `italic` on the `font style`, or maybe different weight other than `normal` on the `font weight`, I believe you will have different result on the preview.

If that wasn't what you meant, please tell me more specific case (the options you selected, etc). Hope this helps you.

Thanks!

**Comment by ivandotv** [comment]
Regarding the  "field interactions > depended field " 
I think if they are the default choice, they should be automatically selected. So the user doesn't gets confused when she selects `normal` and nothing happens.

**Comment by daviedR** [comment]
Yes, we have been considering this manner to be added. But we need to look into the other interactions involved to the fields. Thanks for the input, we would notice you when we have something about this.
